tools/kernel-build-yml.sh

```diff
@@ -13,6 +13,17 @@ if [ "$OS" = "Darwin" ]; then
 fi
 
 KBUILD_BUILD_HOST="eve"
+PREEMPT_RT_SUPPORT="false"
+IMAGE="eve-kernel"
+
+case "$1" in
+--preempt-rt-support)
+    PREEMPT_RT_SUPPORT="true"
+    IMAGE="eve-rt-kernel"
```

Comment:
Does this mean you'll push an image of that name to docker hub? (If so we need to ask IT to create that under lfedge.)

Reply:
I build the image with this name, true. But when the image is pushed - is not clear to me. Do i need to do that explicitly? Will shoot a ticket on lf-edge Jira to create a corresponding docker package.

Reply:
Left the ticket on LFEDGE to create the 'eve-rt-kernel' repo.

Reply:
FWIW the publish workflow is what pushes to docker hub. I haven't checked it only pushes very specific things like lfedge/eve:<tag>, or other containers.